terraform/gcp/security/gcp-org-member-default-service-account-iam-member.yaml

```diff
@@ -0,0 +1,25 @@
+rules:
+    - id: gcp-org-member-default-service-account-iam-member
+      patterns:
+        - pattern: resource
+        - pattern-inside: |
+              resource "google_organization_iam_member" "..." {
+              ...
+              member = $MEMBER
+              ...
+              }
+        - metavariable-regex:
+            metavariable: $MEMBER
+            regex: ((.*)-compute@developer.gserviceaccount.com)

```

Comment:
Same question as above

Reply:
The default service account structure is `PROJECT_NUMBER-compute@developer.gserviceaccount.com` (https://cloud.google.com/compute/docs/access/service-accounts#default_service_account).